incubator/hnc/config/crd/bases/hnc.x-k8s.io_hierarchyconfigurations.yaml

```diff
@@ -14,7 +12,7 @@ spec:
     listKind: HierarchyConfigurationList
     plural: hierarchyconfigurations
     singular: hierarchyconfiguration
-  scope: Namespaced
+  scope: ""
```

Comment:
I think you need to update your version of kubebuilder? See what version Sophie has - I think one of her PRs added the scope.

Reply:
Ah sg. Didn't saw your comment here. I will just update my kubebuilder to solve this issue!